docs/developing.md

```diff
@@ -210,6 +210,33 @@ For example, the Lennard-Jones potential can be parameterized through either the
 %  '''''''''''''
 %  If you define a new class, add new files to autodoc
 
+### Non-bonded methods as implemented in OpenMM
+
+The SMIRNOFF specification describes the contents of a force field, which can be implemented in a number of different ways in different molecular simulation engines.
+The OpenMM implementation provided by the OpenFF Toolkit produces either an `openmm.System` object containing a `openmm.NonbondedForce` object or raises an exception if there is any incompatibility or unimplemented behavior.
+The below table describes these exceptions or which [`NonbondedMethod`](http://docs.openmm.org/latest/userguide/application.html?highlight=ljpme#nonbonded-interactions) is set on the resulting `NonbondedForce`.
+
+| vdw_method | electrostatics_method | periodic | OpenMM Nonbonded method or exception |
+|------------|-----------------------|----------|--------------------------------------|
+| cutoff     | Coulomb               | True     | `IncompatibleParameterError`
+| cutoff     | Coulomb               | False    | `openmm.NonbondedForce.NoCutoff`
+| cutoff     | reaction-field        | True     | `SMIRNOFFSpecError`
+| cutoff     | reaction-field        | False    | `SMIRNOFFSpecUnimplementedError`
+| cutoff     | PME                   | True     | `openmm.NonbondedForce.PME`
+| cutoff     | PME                   | False    | `openmm.NonbondedForce.NoCutoff`
+| LJPME      | Coulomb               | True     | `IncompatibleParameterError`
+| LJPME      | Coulomb               | False    | `openmm.NonbondedForce.NoCutoff`
+| LJPME      | reaction-field        | True     | `IncompatibleParameterError`
+| LJPME      | reaction-field        | False    | `SMIRNOFFSpecError`
+| LJPME      | PME                   | True     | `openmm.NonbondedForce.LJPME`
+| LJPME      | PME                   | False    | `openmm.NonbondedForce.NoCutoff`
+
+Notes:
+* The most commonly-used case (including the Parsley line) is row is the sixth row (cutoff vdW, PME electrostatics, periodic topology)
```

Comment:
This confused me until I realized you meant "sixth row, if you include the header". I think it's clearer to say "fifth".
 
```suggestion
* The most commonly-used case (including the Parsley line) is the fifth row (cutoff vdW, PME electrostatics, periodic topology)
```

Reply:
I agree with Jeff. The header is the zeroth row 😛

Could also consider marking the row itself; making it bold, or adding an asterisk or highlighting the background or something. A simple asterisk would probably be cleanest for Markdown readability.

Reply:
Good catch - I more likely forgot how to count than decide a header row counts as a row